### PR TITLE
AIRFLOW-261 Add more fields for EmailOperator

### PR DESCRIPTION
**Pull request by ajayyadava**
Addresses [AIRFLOW-261](https://issues.apache.org/jira/browse/AIRFLOW-261)

Testing Done:
- Added new Unit test.
- All tests passed on [my travis-ci](https://travis-ci.org/ajayyadava/incubator-airflow/builds/145787612)

### CODE REVIEW

File: airflow/utils/email.py [hunk 1]
```diff
@@ -96,3 +101,15 @@ def send_MIME_email(e_from, e_to, mime_msg, dryrun=False):
         logging.info("Sent an alert email to " + str(e_to))
         s.sendmail(e_from, e_to, mime_msg.as_string())
         s.quit()
+
+
+def get_email_address_list(address_string):
+    if isinstance(address_string, basestring):
+        if ',' in address_string :
```
Comment (skudriashev) [inline airflow/utils/email.py]: Redundant whitespace before `:`.

Comment (ajayyadava) [inline airflow/utils/email.py]: Ah, thanks for pointing it out! Will fix it.

**Comment by ajayyadava** [comment]
I believe the test failures are unrelated.

**Comment by ajayyadava** [comment]
Thanks @bolkedebruin

**Pull request closed without merging**